internal/controllers/machineset/machineset_delete_policy_test.go

```diff
@@ -68,6 +68,17 @@ func TestMachineToDelete(t *testing.T) {
 			},
 		},
 	}
+	healthyCheckConditionFalseMachine := &clusterv1.Machine{
```

Comment:
nit:  `HealthCheckSucceededConditionFalseMachine`?

Reply:
I will fix also this in the new PR